### ISSUE TITLE
Problems caused by a change in 2.5.1

### ISSUE THREAD

**Issue by shouldnotappearcalm** [srikanth-lingala/zip4j#259]
![image](https://user-images.githubusercontent.com/20592210/99685049-1d147e80-2abd-11eb-89bf-c9ee86f01fde.png)

Problems caused by a change in 2.5.1

In this change, when I set charset=UTF-8, new String will use `ZIP_STANDARD_CHARSET`, and the Chinese directory name will be garbled

use 2.6.4

![image](https://user-images.githubusercontent.com/20592210/99685547-aa57d300-2abd-11eb-9a86-dbf0c75d09e4.png)

this is right
![image](https://user-images.githubusercontent.com/20592210/99685714-da9f7180-2abd-11eb-9e49-5a0833bebdaa.png)

This is the compressed package and code I used
[test2.zip](https://github.com/srikanth-lingala/zip4j/files/5567740/test2.zip)

```java
final ZipFile zipFile = new ZipFile("/Users/gzr/dev/test/zip_test/test2.zip");
zipFile.setCharset(StandardCharsets.UTF_8);
final List<FileHeader> fileHeaders = zipFile.getFileHeaders();
```

**Comment by srikanth-lingala** [comment]
Issue fixed. I will include the fix in the next release.

**Comment by shouldnotappearcalm** [comment]
Okay, thank you very much, and look forward to the release of the next release.

**Comment by srikanth-lingala** [comment]
Issue fixed in v2.7.0 which was released today.
